index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+        <link
+    rel="stylesheet"
+    href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+    integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous"    />
+    <div id="page-wrapper">
+    <header id="header">
+        <div class="logo">
+        <img
+            id="header-img"
+            src="logo.png"
+            alt="original trombones logo"
+        />
```

Comment:
Good job but please describe `why` the changes are necessary like `to make the project professional or it a project requirement ` to make the user clear why the changes are being asked.

Reply:
I think it would be better to if you Reduce the size of your logo. Because the size of logo section is supposed to be like the size of logo of the project in the design.

Reply:
I think it would be better to if you Reduce the size of your logo. Because the size of logo section is supposed to be like the size of logo of the project in the design.